Libraries/PeriphDrivers/Source/UART/uart_me17.c

```diff
@@ -263,13 +263,15 @@ int MXC_UART_SetFlowCtrl(mxc_uart_regs_t *uart, mxc_uart_flow_t flowCtrl, int rt
             MXC_GPIO_Config(&gpio_cfg_uart0_flow);
             break;
 
+#if (TARGET_NUM != 78000)
```

Comment:
Are these necessary? Looking at max78000_files.mk, I don't see uart_me17.c getting added to the list of files to compile.

Reply:
Nope.